Hush/Assets/Scripts/Enemies/EnemySword.cs

```diff
@@ -33,11 +32,16 @@ void OnTriggerEnter(Collider other)
 
         #region Public Variables
 
+        public string WeaponType
+        {
+            get => "EnemySword";
+        }
```

Comment:
```suggestion
        public string WeaponType => "EnemySword";
```

Reply:
Why use a getter though instead of a const or just a variable with a default ?

Reply:
nevermind I get it now with the rest!

Reply:
C# is dumb and doesn't allow it.
Interfaces in C# are intended to define the contract that a class will adhere to, not a particular implementation. So to define a property, I have to define it w/ a getter. [StackOverflow explanation](https://stackoverflow.com/questions/2115114/why-cant-c-sharp-interfaces-contain-fields)

Reply:
Yeah it makes sense in the context of the IWeapon it's just the first file I had checked 🥴